srcpkgs/go-ethereum/template

```diff
@@ -10,15 +10,14 @@ go_package="${go_import_path}/cmd/abigen
  ${go_import_path}/cmd/geth
  ${go_import_path}/cmd/puppeth
  ${go_import_path}/cmd/rlpdump
- ${go_import_path}/cmd/wnode
- ${go_import_path}/cmd/swarm"
```

Comment:
`swarm` is split into separate repo https://github.com/ethereum/go-ethereum/pull/19661
I'll package it later when I have a chance.